prow/scripts/cluster-integration/kyma-busola-long-lasting.sh

```diff
@@ -28,13 +28,12 @@ function delete_cluster(){
 
 function provisionCluster() {
     export DOMAIN_NAME=$1
-
-    RESOURCES_PATH=${TEST_INFRA_SOURCES_DIR}/prow/scripts/resources/busola/
+    export DEFINITION_PATH=$2
 
     log::info "Creating cluster: ${DOMAIN_NAME}"
     # create the cluster
     # shellcheck disable=SC2002
-    cat "${RESOURCES_PATH}/cluster-busola.yaml" | envsubst | kubectl create -f -
+    cat "${DEFINITION_PATH}" | envsubst | kubectl create -f -
```

Comment:
Why disable shellcheck when it can be compliant?
```
envsubst < "${DEFINITION_PATH}" | kubectl create -f -
```
Also shouldn't we use `kubectl apply -f -` instead?

Reply:
I changed the line as you suggested.
TBH I disabled shellcheck cause I hate the reverse "<" syntax. But you are right of course....
I think we can leave the "create" argument, this line should fail when cluster already exist (it will not when you use apply)